rpcserver.go

```diff
@@ -2713,16 +2713,26 @@ func (r *rpcServer) AddInvoice(ctx context.Context,
 
 			// Make sure the channel is active.
 			link, err := r.server.htlcSwitch.GetLink(chanPoint)
-			if err != nil {
-				rpcsLog.Errorf("Unable to get link for "+
-					"channel %v: %v", chanPoint, err)
+			if err != nil || !link.EligibleToForward() {
```

Comment:
I think we should keep the existing way of logging the error.

Reply:
Fixed.